agent/src/main/java/com/microsoft/hydralab/agent/runner/espresso/EspressoTestInfoProcessorListener.java

```diff
@@ -71,6 +74,9 @@ public void startRecording(int maxTime) {
         final String initializing = "Initializing";
         testRunDeviceOrchestrator.setRunningTestName(testRunDevice, initializing);
         testRun.addNewTimeTag(initializing, 0);
+        if (testTask.isEnableNetworkMonitor()) {
```

Comment:
In the future we are going to disable the screen recording for long-time test run. Going forward, the logics of starting VPN should be removed from `startRecording()`. cc @zhou9584

Reply:
yes